admin/services/pupil-register-caching.service.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const featureToggles = require('feature-toggles')
+
+const pupilRegisterV2Service = require('./pupil-register-v2.service')
+const pupilRegisterService = require('./pupil-register.service')
+const redisCacheService = require('./data-access/redis-cache.service')
+
+const pupilRegisterCachingService = {
```

Comment:
i don't believe we need a dedicated pupil register cache, all the logic should be encapsulated in the pupil register service.

Reply:
@GuyHarwood yes, I removed it.